graphene_django/filter/utils.py

```diff
@@ -18,9 +18,16 @@ def get_filtering_args_from_filterset(filterset_class, type):
         if name in filterset_class.declared_filters:
             form_field = filter_field.field
         else:
-            field_name = name.split("__", 1)[0]
-
-            if hasattr(model, field_name):
+            try:
+                field_name, filter_type = name.split("__", 1)
```

Comment:
Should use `rsplit` instead: https://github.com/graphql-python/graphene-django/issues/750#issuecomment-527541331

Reply:
Thanks @zbyte64 . Fixed